config/config-builder/src/validator_config.rs

```diff
@@ -30,6 +30,14 @@ pub struct ValidatorConfig {
     build_waypoint: bool,
     bootstrap: NetworkAddress,
     index: usize,
+    key_manager_json_rpc_endpoint: Option<String>,
+    key_manager_rotation_period_secs: Option<u64>,
+    key_manager_sleep_period_secs: Option<u64>,
+    key_manager_storage_backend: Option<String>,
+    key_manager_storage_host: Option<String>,
+    key_manager_storage_token: Option<String>,
+    key_manager_storage_namespace: Option<String>,
+    key_manager_txn_expiration_secs: Option<u64>,
```

Comment:
why is this part of ValidatorConfig? Wouldn't it make more sense for this to be in its own "Config"?

Reply:
Thanks! Moved to it's own config 😄